lightning/src/offers/signer.rs

```diff
@@ -38,23 +37,25 @@ const DERIVED_METADATA_AND_KEYS_HMAC_INPUT: &[u8; 16] = &[2; 16];
 const WITHOUT_ENCRYPTED_PAYMENT_ID_HMAC_INPUT: &[u8; 16] = &[3; 16];
 const WITH_ENCRYPTED_PAYMENT_ID_HMAC_INPUT: &[u8; 16] = &[4; 16];
 
-// HMAC input for a `PaymentId`. The HMAC is used in `OffersContext::OutboundPayment`.
-const OFFER_PAYMENT_ID_HMAC_INPUT: &[u8; 16] = &[5; 16];
-// HMAC input for a `PaymentId`. The HMAC is used in `AsyncPaymentsContext::OutboundPayment`.
-#[cfg(async_payments)]
-const ASYNC_PAYMENT_ID_HMAC_INPUT: &[u8; 16] = &[6; 16];
-
-// HMAC input for a `PaymentHash`. The HMAC is used in `OffersContext::InboundPayment`.
-const PAYMENT_HASH_HMAC_INPUT: &[u8; 16] = &[7; 16];
+// NOTE:
+// The following `HMAC_INPUT` constants were previously used for authenticating fields
+// in `OffersContext` and `AsyncPaymentsContext`, but were removed in LDK v0.1.X with
```

Comment:
@TheBlueMatt

which version will this merged?

Reply:
Done! [pr3917.07](https://github.com/shaavan/rust-lightning/commits/pr3917.07)
Thanks for the pointer! ✨